friendly-business/style-editor.scss

```diff
@@ -6,6 +6,9 @@ Friendly Business Editor Styles
 
 @import "sass/variables-site/variables-site";
 @import "sass/mixins/mixins-master";
+@import "sass/site/header/site-header";
+@import "sass/navigation/menu-main-navigation";
```

Comment:
We can leave this for a follow up but I don't think the navigation styling here is being picked up. Note the list style, vs Andy's summary screenshot

<img width="1322" alt="Screen Shot 2019-07-17 at 10 46 46 AM" src="https://user-images.githubusercontent.com/1270189/61400415-0df83d80-a885-11e9-83b7-413cbc07414e.png">


We probably also would like to pull in `sass/navigation/navigation` instead for the other menu types

Reply:
I can't believe I missed that! I had a hunch based on some previous conversations, but it turns out I did not have a primary menu created in the customizer on my site. Once I made a primary menu there, the style looks correct in the template editor:

<img width="1630" alt="Screen Shot 2019-07-17 at 11 28 07 AM" src="https://user-images.githubusercontent.com/6265975/61400828-f8374800-a885-11e9-9513-500175952588.png">

Reply:
Good catch @noahtallen! Adding a main menu did the trick

<img width="1017" alt="Screen Shot 2019-07-17 at 11 31 35 AM" src="https://user-images.githubusercontent.com/1270189/61402719-93312180-a888-11e9-89d6-74cf87d4575e.png">

Reply:
I think we're tracking it here: https://github.com/Automattic/wp-calypso/issues/34429 I remember it came up in the modern business PR

Reply:
Perfect ✨